### ISSUE TITLE
Tester

### ISSUE THREAD

**Issue by GA33O** [jemsz95/global-game-jam-2018#1]
Buenas tardes, puedo testear su juego? Estoy empezando en esto y me ayudaría mucho para practicar este tipo de cosas, no se si pueda pedirlo por aquí, no vi alguna manera de contacto ya sea FB o Twitter. Gracias

**Comment by jemsz95** [comment]
Claro! Por mi no hay problema. Cualquier pregunta estoy aqui para ayudar.

**Comment by GA33O** [comment]
Solo una cosa por el momento, en el menú, hay un apartado llamado (Transmit) le doy clic y no pasa nada, debería abrir algo?
